### DISCUSSION TITLE
Team C - Course 1

### DISCUSSION THREAD

**Discussion by fregataa** [lablup/contribution-academy-2021#38] [Show and tell]
Team C - Course 1 Backend.ai 사용해보기

## executed time (for training only)
![image](https://user-images.githubusercontent.com/44239739/128989839-bf832d83-189a-45b6-9097-64818ff7e362.png)

## accuracy
![image](https://user-images.githubusercontent.com/44239739/128989926-3461a13a-c5a4-47d6-b0cf-f6fcdad7656b.png)

## total allocated resources:
![image](https://user-images.githubusercontent.com/44239739/128989975-147175a8-357b-4e28-9982-a46db9a24ae3.png)

---
@lizable 
C팀 Course 1 완료했습니다

**Comment by yujung7768903** [comment]
#### executed time
![image](https://user-images.githubusercontent.com/68562176/128991311-894b18ef-175d-4865-969b-5043d4a46345.png)

#### accuracy
![image](https://user-images.githubusercontent.com/68562176/128991364-fd12ad27-dca1-4c10-a379-b17ff61104a0.png)

#### total allocated resources
![image](https://user-images.githubusercontent.com/68562176/128991637-a5cfeba6-5ef1-4240-87fc-8511cfc89d0b.jpeg)

**Comment by jinseo-ok** [comment]
![image](https://user-images.githubusercontent.com/48677363/129023736-ba81b81c-4c58-47d2-b4fc-c7f3b5840239.png)

**Comment by euni00** [comment]
- Executed time

<img width="796" alt="Executed time" src="https://user-images.githubusercontent.com/82070770/129038173-ce8a2f7d-4ab6-4e70-9bcb-afe3c8c49632.png">


- Accuracy

<img width="533" alt="accuracy" src="https://user-images.githubusercontent.com/82070770/129036798-3fd6f450-26ea-4015-8593-378d0dbbaf09.png">


- Total allocated resources

<img width="1126" alt="total allocated resources" src="https://user-images.githubusercontent.com/82070770/129036889-92b1b5d9-b047-4507-a9d7-8a6c9ef5c53c.png">
